static_src/components/panel_row.jsx

```diff
@@ -29,9 +27,10 @@ export default class PanelRow extends React.Component {
   render() {
     const props = this.props;
     const styleClass = props.styleClass && `panel-row-${this.props.styleClass}`;
+    const mainClass = props.styleClass !== 'boxed' && 'panel-row';
 
     return (
-      <div className={ this.styler('panel-row', styleClass) }>
+      <div className={ this.styler(mainClass, styleClass) }>
```

Comment:
Because the boxed component is almost becoming a separate thing. I think I want to split it out into it's own component. Again, I'm waiting on the PL work because some of the work on the overview component might make sense. I actually want to sit down and refactor the whole panel component before we end.